src/components/DocPage/DocPage.tsx

```diff
@@ -376,7 +376,7 @@ class DocPage extends React.Component<DocPageInnerProps, DocPageState> {
     private renderAuthor(onlyAuthor: boolean) {
         const {meta, lang} = this.props;
 
-        if (!meta?.author) {
+        if (!meta?.author || typeof meta?.author !== 'object') {
```

Comment:
According to typings, it's either object or undefined, it's not true?

https://github.com/yandex-cloud/docs-components/blob/518599d8c025ed9a2437b3330751fddd88485cd4/src/models/index.ts#L86

https://github.com/yandex-cloud/docs-components/blob/518599d8c025ed9a2437b3330751fddd88485cd4/src/models/index.ts#L122

Reply:
Sounds strange for me...

Reply:
I think need add unknown type, cause metadata may contain any data.